translations/en.json

```diff
@@ -1928,7 +1928,7 @@
   "password-creation-max-length-hint": "Maximum 100 characters",
   "password-creation-placeholder-1": "Type password",
   "password-creation-placeholder-2": "Repeat password",
```

Comment:
Thank you for paying attention to these small details @ilmotta 

By solving these small UX issues we improve the quality of our app. I saw this weird animation and forgot to fix it 🤦‍♂️, I'll fix it 👍

Reply:
Lot of things to say about this fix.

Basically, this bug happens because of the `floating-button-page` component when a `header` along with `:content-avoid-keyboard?` are provided.

That component uses the `on-layout` RN callback to properly move the content when the keyboard appears and there's a header.

I fixed it by providing the already calculated header height. On RN 0.76 they added a hook to fix the `on-layout` issue, so this could behave better once we upgrade our version, please see: https://reactnative.dev/blog/2024/10/23/the-new-architecture-is-here#uselayouteffect

Here's a video of the new behavior:

https://github.com/user-attachments/assets/d733f2d3-a4df-448d-8ac2-ae4d2c50c268

And here's a slowmo video (it's just me manually moving the video's time handle :laughing: ) but you can appreciate the fix:

https://github.com/user-attachments/assets/3c63e3ed-e39c-42b5-b1fd-541dfe5eca38

Reply:
Oh nice! Sometimes little UI bugs are the trickiest 🐞